jobbergate-cli/jobbergate_cli/config.py

```diff
@@ -28,6 +28,7 @@ class Settings(BaseSettings):
 
     SENTRY_DSN: Optional[str]
     SENTRY_TRACE_RATE: float = Field(1.0, gt=0.0, le=1.0)
+    SENTRY_ENV: str = "local"
```

Comment:
A comment in the beginning of the block would be good to maintain consistency:

```suggestion
    # Sentry <resume functionality here>
    SENTRY_DSN: Optional[str]
    SENTRY_TRACE_RATE: float = Field(1.0, gt=0.0, le=1.0)
    SENTRY_ENV: str = "local"
```

I'm just not sure about the exacly sentence, what do you think?

Reply:
sure, sounds good